Libraries/LibJS/AST.h

```diff
@@ -120,13 +120,17 @@ class ScopeNode : public Statement {
     void add_variables(NonnullRefPtrVector<VariableDeclaration>);
     const NonnullRefPtrVector<VariableDeclaration>& variables() const { return m_variables; }
 
+    bool is_strict_mode() const { return m_is_strict_mode; }
```

Comment:
Let's call this `in_strict_mode()` so it reads more naturally.
And `m_strict_mode` for the member.

Same comment repeated for all occurrences of the pattern :)